includes/Core/Util/Migration_1_45_0.php

```diff
@@ -0,0 +1,120 @@
+<?php
+/**
+ * Migration for 1.45.0
+ *
+ * @package   Google\Site_Kit\Core\Util
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Util;
+
+use Google\Site_Kit\Context;
+use Google\Site_Kit\Core\Authentication\Authentication;
+use Google\Site_Kit\Core\Authentication\Google_Proxy;
+use Google\Site_Kit\Core\Authentication\Profile;
+use Google\Site_Kit\Core\Authentication\Verification_File;
+use Google\Site_Kit\Core\Authentication\Verification_Meta;
+use Google\Site_Kit\Core\Permissions\Permissions;
+use Google\Site_Kit\Core\Storage\Options;
+use Google\Site_Kit\Core\Storage\User_Options;
+use WP_User;
+use WP_Error;
+
+/**
+ * Class Migration_1_45_0
+ *
+ * @since 1.45.0
+ * @access private
+ * @ignore
+ */
+class Migration_1_45_0 {
```

Comment:
We need to instantiate this class in the Plugins class as we do it for other migration classes: https://github.com/google/site-kit-wp/blob/740fa3262c465dafd097a7b8d4100bf68c6a52c4/includes/Plugin.php#L194-L195

Reply:
I thought I had done this... https://github.com/google/site-kit-wp/pull/4305/commits/50a92f41a8c0e6ac53fc8667ce803c739f6a0e37